### ISSUE TITLE
Missing weight files?

### ISSUE THREAD

**Issue by munsanje** [yhenon/keras-frcnn#6]
Where can we find the weights specified in config.py?

**Comment by yhenon** [comment]
Keras provides the weights in the applications (see here: https://github.com/fchollet/keras/blob/master/keras/applications/resnet50.py). I've added a print statement (commit 93d837fd5f7f894674f8bb722748b75d45ee3f94) to make this clearer. If the weight files are not found, it will now print:

```
     Could not load pretrained model weights. Weights can be found at https://github.com/fchollet/deep-learning-models/releases/download/v0.2/resnet50_weights_th_dim_ordering_th_kernels_notop.h5 and https://github.com/fchollet/deep-learning-models/releases/download/v0.2/resnet50_weights_tf_dim_ordering_tf_kernels_notop.h5
```

Please feel free to reopen this issue if any problems concerning loading the weights remain.

**Comment by munsanje** [comment]
Thanks. Is it normal to have 'nan' loss initially?

**Comment by yhenon** [comment]
That is not normal. Could you tell me what backend you are using? Have you made any other changes?

**Comment by munsanje** [comment]
I specified the backend 'tf' in the parameters to data generator as mentioned in my other issue. I haven't changed anything else. I'm using tensorflow.

**Comment by munsanje** [comment]
Hey, tried it out again on a more powerful/faster machine and the loss is no longer nan. Guess it was an issue of training speed. Cheers!
